uiowa/uiowa
# Align Center for Embedded Video

resolves #2051

## To Test
- Non-collegiate sites using entity browser.
- Embed a video with align center set in uids_base using entity browser. See that the video is centered.
